### PR TITLE
Tidy SDP atoms from CVXQUAD

### PR DESCRIPTION
**Pull request by odow**
Tidying and tests related to https://github.com/jump-dev/Convex.jl/pull/604

### CODE REVIEW

File: src/atoms/sdp_cone/QuantumRelativeEntropyAtom.jl [hunk 1]
```diff
@@ -39,22 +48,60 @@ mutable struct QuantumRelativeEntropy1Atom <: AbstractExpr
...
+) where {T}
+    A, B = atom.children[1], atom.children[2]
+    add_constraint!(context, A ⪰ 0)
+    add_constraint!(context, B ⪰ 0)
+    I = Matrix(one(T) * LinearAlgebra.I(size(A, 1)))
+    m, k, e = atom.m, atom.k, vec(I)
+    τ = Variable()
+    add_constraint!(
+        context,
+        τ in  RelativeEntropyEpiCone(kron(A, I), kron(I, conj(B)), m, k, e),
```
Comment (odow) [inline src/atoms/sdp_cone/QuantumRelativeEntropyAtom.jl:100]:
```suggestion
        τ in RelativeEntropyEpiCone(kron(A, I), kron(I, conj(B)), m, k, e),
```

File: src/atoms/sdp_cone/QuantumEntropyAtom.jl [hunk 2]
```diff
@@ -5,28 +5,32 @@
 # in the LICENSE.md file or at https://opensource.org/licenses/MIT.
 
 """
-quantum_entropy returns -LinearAlgebra.tr(X*log(X)) where X is a positive semidefinite.
+    QuantumEntropyAtom(X::AbstractExpr, m::Integer, k::Integer)
+
+quantum_entropy returns -LinearAlgebra.tr(X*log(X)) where X is a positive
```
Comment (blegat) [inline src/atoms/sdp_cone/QuantumEntropyAtom.jl:10]:
```suggestion
`quantum_entropy` returns `-LinearAlgebra.tr(X*log(X))` where `X` is a positive
```

File: src/atoms/sdp_cone/QuantumEntropyAtom.jl [hunk 3]
```diff
@@ -5,28 +5,32 @@
...
+
 Note this function uses logarithm base e, not base 2, so return value is in
 units of nats, not bits.
 
 Quantum entropy is concave. This function implements the semidefinite
 programming approximation given in the reference below.  Parameters m and k
-control the accuracy of this approximation: m is the number of quadrature
-nodes to use and k the number of square-roots to take. See reference for
-more details.
+control the accuracy of this approximation: m is the number of quadrature nodes
```
Comment (blegat) [inline src/atoms/sdp_cone/QuantumEntropyAtom.jl:18]:
```suggestion
control the accuracy of this approximation: `m` is the number of quadrature nodes
```

File: src/atoms/sdp_cone/QuantumEntropyAtom.jl [hunk 4]
```diff
@@ -5,28 +5,32 @@
...
+to use and k the number of square-roots to take. See reference for more details.
```
Comment (blegat) [inline src/atoms/sdp_cone/QuantumEntropyAtom.jl:19]:
```suggestion
to use and `k` the number of square-roots to take. See reference for more details.
```

File: src/atoms/sdp_cone/QuantumRelativeEntropyAtom.jl [hunk 5]
```diff
@@ -5,27 +5,36 @@
...
+        m::Integer,
+        k::Integer,
+    )
 
-Quantum relative entropy is convex (jointly) in (A,B). This function
-implements the semidefinite programming approximation given in the reference
-below.  Parameters m and k control the accuracy of this approximation: m is
-the number of quadrature nodes to use and k the number of square-roots to
-take. See reference for more details.
+quantum_relative_entropy returns LinearAlgebra.tr(A*(log(A)-log(B))) where A and
```
Comment (blegat) [inline src/atoms/sdp_cone/QuantumRelativeEntropyAtom.jl:15]:
```suggestion
`quantum_relative_entropy` returns `LinearAlgebra.tr(A*(log(A)-log(B)))` where `A` and
```

File: src/atoms/sdp_cone/QuantumRelativeEntropyAtom.jl [hunk 6]
```diff
@@ -5,27 +5,36 @@
...
+B are positive semidefinite matrices.
```
Comment (blegat) [inline src/atoms/sdp_cone/QuantumRelativeEntropyAtom.jl:16]:
```suggestion
`B` are positive semidefinite matrices.
```

**Comment by odow** [comment]
Yeah just moving things. I'll fix the docstring math as part of #610

**Pull request merged**